behat/tests/bootstrap/v2/reporting/sections/ActionsSectionTrait.php

```diff
@@ -0,0 +1,143 @@
+<?php declare(strict_types=1);
+
+namespace DigidepsBehat\v2\Reporting\Sections;
+
+trait ActionsSectionTrait
+{
+    private int $answeredYes = 0;
+    private int $answeredNo = 0;
+    private array $comments = [];
+
+    /**
+     * @Given I view the actions report section
+     */
+    public function iViewActionsSection()
+    {
+        $activeReportId = $this->loggedInUserDetails->getCurrentReportId();
+        $reportSectionUrl = sprintf(self::REPORT_SECTION_ENDPOINT, $activeReportId, 'actions');
+        $this->visitPath($reportSectionUrl);
+    }
+
+    /**
+     * @Given I view and start the actions report section
+     */
+    public function iViewAndStartActionsSection()
+    {
+        $this->iViewActionsSection();
+        $this->clickLink('Start actions');
+    }
+
+    /**
+     * @Given I choose no and save on financial decision actions section
+     */
+    public function iChooseNoOnFinancialDecisionActionsSection1()
+    {
+        $this->fillInActionsForm(
+            'no',
+            'action[doYouExpectFinancialDecisions]'
+        );
+    }
+
+    /**
+     * @Given I choose yes and save on financial decision actions section
+     */
+    public function iChooseYesOnFinancialDecisionActionsSection1()
+    {
+        $this->fillInActionsForm(
+            'yes',
+            'action[doYouExpectFinancialDecisions]',
+            'first comment',
+            'action[doYouExpectFinancialDecisionsDetails]'
+        );
+    }
+
+    /**
+     * @Given I choose no and save on concerns actions section
+     */
+    public function iChooseNoOnDoYouHaveConcernsActionsSection()
+    {
+        $this->fillInActionsForm(
+            'no',
+            'action[doYouHaveConcerns]',
+        );
+    }
+
+    /**
+     * @Given I choose yes and save on concerns actions section
+     */
+    public function iChooseYesOnDoYouHaveConcernsActionsSection()
+    {
+        $this->fillInActionsForm(
+            'yes',
+            'action[doYouHaveConcerns]',
+            'second comment',
+            'action[doYouHaveConcernsDetails]'
+        );
+    }
+    public function fillInActionsForm($answer, $actionName, $comment = null, $commentName = null)
+    {
+        $this->selectOption($actionName, $answer);
+        if ($answer === "no") {
+            $this->answeredNo += 1;
+        } elseif ($answer === "yes") {
+            $this->answeredYes += 1;
+        }
```

Comment:
Love this - was trying to work out how to assert on multiple yes and no responses ⭐

Reply:
I think we can actually do this better. I didn't want to waste further time in this ticket but I've got some improvements to try on next one of these.